### PR TITLE
Placeholder PR for JOSS diff

### PR DESCRIPTION
**Pull request by wildsm**

### CODE REVIEW

File: docs/papers/joss/paper.md [hunk 1]
```diff
@@ -71,44 +71,72 @@ Some crucial considerations relevant to these packages include:
...
+libEnsemble attempts to minimize required dependencies while still supporting optional
+communication mechanisms and application schedulers. For example, for the vast majority of users
+Python's multiprocessing module is sufficient for parallelism, but for those seeking  
+higher or more complex scaling, simply installing MPI and mpi4py
+allows libEnsemble to run with a MPI runtime without adjusting any code. Most users are likewise satisfied
+scheduling application instances to the same allocation as a libEnsemble instance; but an additional
+pip-install of Balsam and one-line adjust are sufficient for libEnsemble to schedule applications across
+the internet.
+
+<!--- JLN: The below paragraph probably needs some rearranging. I think users don't typically seek
```
Comment (shuds13) [inline docs/papers/joss/paper.md:105]: I dont think I suggested the user wants to run a database, but many workflow tools require it.

Comment (shuds13) [reply docs/papers/joss/paper.md:105]: I prefer my original wording for the most part

File: docs/papers/joss/paper.md [hunk 2]
```diff
@@ -71,44 +71,72 @@ Some crucial considerations relevant to these packages include:
...
+
 For cases where autodetection is insufficient, the user can supply platform information or the name
 of a known system via scripts or an environment variable.
 
-By default, libEnsemble divides available compute resources amongst workers. However, when simulation
-parameters are created, the number of processes and GPUs can also be specified for each simulation.
-Combined with the portability features, this makes it very simple to transfer user scripts between platforms.
+-->
 
-The close coupling between the libEnsemble generator and simulators enables the generator to perform tasks
```
Comment (shuds13) [inline docs/papers/joss/paper.md]: Why removed?

File: docs/papers/joss/paper.md [hunk 3]
```diff
@@ -71,44 +71,72 @@ Some crucial considerations relevant to these packages include:
 
 [merge sim/gen with this?]
 
-libEnsemble stands out primarily through its generator-simulator paradigm, which eliminates the need
-for users to explicitly define task dependencies. Instead, it emphasizes data dependencies between
-customizable Python user functions. This modular design also lends itself to exploiting the large
-library of example user functions that are provided with libEnsemble, maximizing code reuse. For
```
Comment (shuds13) [inline docs/papers/joss/paper.md]: I wanted to high-light code reuse, Why remove?

File: docs/papers/joss/paper.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+such as asynchronously receiving results, updating models, and canceling previously initiated simulations.
+Simulations that are already running can be terminated and resources recovered. This is more flexible
+compared to other packages, where the generation of simulations is external to the dispatch of a batch 
+of simulations.
+
+libEnsemble attempts to minimize required dependencies while still supporting optional
+communication mechanisms and application schedulers. For example, for the vast majority of users
+Python's multiprocessing module is sufficient for parallelism, but for those seeking  
+higher or more complex scaling, simply installing MPI and mpi4py
+allows libEnsemble to run with a MPI runtime without adjusting any code. Most users are likewise satisfied
```
Comment (jmlarson1) [inline docs/papers/joss/paper.md:100]: The statement "run with a MPI runtime" is unclear.

File: docs/papers/joss/paper.md (hunk 1 above)
Comment (jlnav) [reply docs/papers/joss/paper.md:105]: Ahh, I see, I misunderstood. Feel free to change it back!

**Comment by wildsm** [comment]
FYI- I was not thinking of ever merging this branch into main. We should discuss later.

**Comment by jmlarson1** [comment]
Closing after acceptance: https://doi.org/10.21105/joss.06031

**Pull request closed without merging**